lte/gateway/c/oai/tasks/ngap/ngap_amf_itti_messaging.c

```diff
@@ -125,6 +125,9 @@ void ngap_amf_itti_ngap_initial_ue_message(
   NGAP_INITIAL_UE_MESSAGE(message_p).m5g_rrc_establishment_cause =
       rrc_cause + 1;
 
+  OAILOG_INFO(LOG_NGAP, "ue context request recvd at ngap : %d", ue_ctx_req);
+  NGAP_INITIAL_UE_MESSAGE(message_p).ue_context_request = ue_ctx_req;
```

Comment:
what is the purpose of this ue_ctx_req? why is it defined as long?

Reply:
- asn decoder decoding UEContextRequest as long type 
vagrant@magma-dev:~/build/c/oai/tasks/ngap/r16$ grep Ngap_UEContextRequest_t *.h
Ngap_UEContextRequest.h:typedef **long**     Ngap_UEContextRequest_t;

**ngap_amf_itti_ngap_initial_ue_message** API sends itti_ngap_initial_ue_message_s  to AMF task.  
 **ue_context_request** place holder where ngap fill this parameter and pass to AMF.